src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/PushHookProcessor.java

```diff
@@ -94,12 +95,25 @@ public boolean isMatch(@NonNull SCMNavigator navigator) {
                                 return false;
                             }
                             BitbucketSCMNavigator bbNav = (BitbucketSCMNavigator) navigator;
+                            if(!isProjectKeyMatch(bbNav.getProjectKey())){
```

Comment:
format code in proper way.
```suggestion
                            if (!isProjectKeyMatch(bbNav.getProjectKey())) {
```

Reply:
Fixed.